client/package.json

```diff
@@ -64,7 +64,7 @@
     "webpack": "^1.12.8"
   },
   "devDependencies": {
-    "babel-eslint": "^4.1.5",
+    "babel-eslint": "^5.0.0-beta4",
```

Comment:
@justin808 version `5.0.0-beta4` uses Babel 6. Maybe this is causing some conflicts? https://github.com/babel/babel-eslint/commit/a4c09a838cdc4b00b3d754b742a09aeb1b8258c1